src-docs/src/views/schema/schema.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import React from 'react';
+
+import { OuiSchemaItem } from '../../../../src/components/schema';
+
+export default () => (
+  <div>
```

Comment:
What's the motivation for using `<div>` and `<br />`? Did you consider using `React.Fragment`/`OuiPanel` and `OuiSpacer` respectively?

Reply:
Will update